examples/statics/kibana-links.yaml

```
@@ -0,0 +1,51 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kibana-links
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: kibana-links
+    busola.io/extension: statics
+    busola.io/extension-version: '0.5'
+data:
+  injections: |-
+    - name: "Logs for namespace"
```

```suggestion
    - name: "Logs for Namespace"
```
should we capitalize it here as in our docs?